pkg/collector/collector.go

```diff
@@ -545,6 +545,20 @@ func (c *Collector) collectCGroups(o *CapacityObjects, ch chan<- prometheus.Metr
 				}
 				metric.Values[l] = cpuValues
 
+				// get the pre-computed average
+				name = MetricName{
+					Source:        "avg_" + src,
```

Comment:
slightly concerned about this potentially causing source names that are not declared anywhere else, maybe we could modify `getCGroupMetricSource` to return both the metric and its average?